error: 

   ┌── tests/move_check/naming/unresolved_type_no_args.move:2:22 ───
   │
 2 │     struct Mine { f: A }
   │                      ^ Could not resolve type name: 'A'
   │

error: 

   ┌── tests/move_check/naming/unresolved_type_no_args.move:3:12 ───
   │
 3 │     foo(x: S): G {
   │            ^ Could not resolve type name: 'S'
   │

error: 

   ┌── tests/move_check/naming/unresolved_type_no_args.move:3:16 ───
   │
 3 │     foo(x: S): G {
   │                ^ Could not resolve type name: 'G'
   │

error: 

   ┌── tests/move_check/naming/unresolved_type_no_args.move:4:16 ───
   │
 4 │         let _: B = (0: P);
   │                ^ Could not resolve type name: 'B'
   │

error: 

   ┌── tests/move_check/naming/unresolved_type_no_args.move:4:24 ───
   │
 4 │         let _: B = (0: P);
   │                        ^ Could not resolve type name: 'P'
   │

